README.md

```diff
@@ -508,6 +508,9 @@ The following are optional:
 * `launch_timeout`: How long to wait for a stack to be launched, in seconds.
   If unset, the global timeout will be used.
 
+* `delete_age`: Delete stacks that haven't been resumed in this many seconds.
+  Overrides the globally defined setting.
```

Comment:
Here, you probably also want to explain _why_ this is expressed in seconds, not days.